gateway/1.6/modules/ROOT/pages/policies-included-header-injection.adoc

```diff
@@ -91,11 +89,6 @@ In Local Mode, you apply the Header Injection policy to your API via declarative
 
 |===
 
-[NOTE]
-====
-`inboundHeaders` and `outboundHeaders` can not both be empty.
```

Comment:
Hey, was this an intentional deletion? I'm a little confused if this is related to the envoy policies or not.